web/packages/teleterm/src/ui/TopBar/MoreOptions.tsx

```diff
@@ -64,6 +64,13 @@ function useMenuItems(): MenuItem[] {
   const isSearchBarEnabled = configService.get('feature.searchBar').value;
 
   const menuItems: MenuItem[] = [
+    {
+      title: 'Open new terminal',
+      isVisible: isSearchBarEnabled,
+      Icon: icons.Terminal,
+      keyboardShortcutAction: 'newTerminalTab',
+      onNavigate: openTerminalTab,
+    },
```

Comment:
What do you think about adding a separator between 'Open new terminal' and 'Open config file'?
It seems to me that 'Open config file', 'Install tsh in PATH' and 'Remove tsh from PATH' can be considered as a logical group.

Reply:
I'm not sure about it, how does opening a config file relate to installing tsh in PATH? If anything, opening a new terminal could be grouped with the two tsh items. But then we'd also have to make the separator not show up on platforms other than macOS since they don't have those extra tsh items.

Reply:
>how does opening a config file relate to installing tsh in PATH?

I think about it as config-related stuff, 'Open config file' configures Connect, installing tsh in PATH configures your OS. But it is just an idea, I'm fine with merging it as is.